src/components/PostDetails/PostDetails.tsx

```diff
@@ -1,45 +1,110 @@
-import React from 'react';
+/* eslint-disable no-console */
+/* eslint-disable padded-blocks */
+import React, { useEffect, useState } from 'react';
 import { NewCommentForm } from '../NewCommentForm';
+import { getPostDetails, getPostComments } from '../../api/posts';
 import './PostDetails.scss';
 
-export const PostDetails: React.FC = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+type Props = {
+  postId: number;
+};
+
+type Comment = {
+  id: number;
+  title: string;
+  body: string;
+};
+
+export const PostDetails: React.FC <Props> = ({ postId }) => {
+  const [details, setDetails] = useState({
+    title: '',
+  });
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [commentsVisible, setCommentsVisible] = useState(true);
+  const [checkIfCommentWasAdded, setCheckIfCommentWasAdded] = useState(false);
```

Comment:
This variable is redundant here
You should call getPostComments function after adding some comment